src/styles/settings.css

```diff
@@ -0,0 +1,13 @@
+.html {
+  box-sizing: border-box;
+
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Great job! Some small improvements on my opinion would be: 
delete box-sizing in html as it's applied to all elements anyway, and set standard font-size and font-color in html for minimizing code in classes)

Reply:
Why do I use box sizing this way?  Instead of the usual * {...}, because some libraries, according to the Swiper type, use box-sizing: content-box (e.g. the swiper-wrapper class) and for the library to display as as the developer intended, she needs to leave box-sizing, specified by the developer. So I use imitation. 
Set standard font-size and font-color in html for minimizing code in classes ,it's true, will change this now